### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Jeedoli**
main <- dev merge

### CODE REVIEW

**Comment by bamjun** [comment]
good

File: django/a_apis/models/products.py [hunk 1]
```diff
@@ -40,15 +40,17 @@ def __str__(self):
 class ProductContents(CommonModel):
     contents_id = models.IntegerField(primary_key=True, verbose_name="이미지/동영상 ID")
     img_url = models.FileField(upload_to="img/", verbose_name="이미지 URL")
-    video_url = models.FileField(upload_to="video/", verbose_name="동영상 URL")
+    video_url = models.FileField(
+        upload_to="video/", verbose_name="동영상 URL", blank=True, null=True
```
Comment (bamjun) [inline django/a_apis/models/products.py:44]: good

**Comment by Ju-MINJAE** [comment]
확인했습니다

**Review by Ju-MINJAE — COMMENTED** [review]
캬 ~

**Pull request merged**